sdk/test-utils/recorder-v2/package.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "@azure/test-utils-recorder-v2",
+  "version": "1.0.0",
```

Comment:
It's weird to have `recorder-v2` be version `1.0.0`. Maybe the recorder could be called `recorder-new` or something like that.

Reply:
Just a random, will surely change.